code/datums/status_effects/debuffs/fire_stacks.dm

```diff
@@ -267,11 +267,7 @@
 	cache_stacks()
 	update_overlay()
 	update_particles()
-	if(!iscarbon(owner))
-		return
-
 	for(var/obj/item/equipped in owner.get_equipped_items())
-		equipped.wash(CLEAN_TYPE_ACID)
 		equipped.extinguish()
```

Comment:
also why

Reply:
I'm not sure it had existed in the first place. This behavior is probably wanted for other mobs that have equipped items.